index.html

```diff
@@ -59,17 +59,30 @@ <h1 class="text-center" id="workWithUs">Work with us</h1>
           <div class="card-info">
             <h4 class="cardTitle">ORGANIZATIONS</h4>
             <h2 class="cardText">Are you a Chicagoland nonprofit that needs some tech help?</h2>
+<<<<<<< Updated upstream
```

Comment:
Please remove these stash markers

Reply:
I think these slash markers were created by GitHub Desktop.

Reply:
If there is anything else I am doing wrong, let me know.

Reply:
Can you please delete the markers from the file?